yapapi/golem.py

```diff
@@ -113,19 +115,41 @@ def subnet_tag(self) -> Optional[str]:
         """Return the name of the subnet, or `None` if it is not set."""
         return self._engine.subnet_tag
 
-    async def __aenter__(self) -> "Golem":
+    @property
+    def operative(self) -> bool:
```

Comment:
The name of this property is somewhat vague, at least to me. How about we rename this to just `started`?

Reply:
This property is essentially "There was a `start` and there was no `stop` after it".

I'm not sure if this is "started" - if you start and that stop something it might be both `started` and `stopped` (and that might be different that not having started at all).

Maybe "active" would be better? Or "running"?

Reply:
I'm fine with either... honestly